spec/rubocop/cop/internal_affairs/dynamic_predicate_name_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::InternalAffairs::DynamicPredicateName, :config do
+  let(:transpose_configure_cop_name) { 'Naming/PredicateName' }
+
+  subject(:cop) { described_class.new(config) }
+
+  context 'with blacklisted prefixes' do
+    let(:cop_config) do
+      { 'NamePrefix' => %w[has_ is_],
+        'NamePrefixBlacklist' => %w[has_ is_] }
+    end
+
+    %w[has is].each do |prefix|
```

Comment:
I prefer [DAMP](https://stackoverflow.com/a/11837973/1520557) tests. Especially since using Ruby loops in the tests makes it impossible to use the `#expect_offense` matcher. 🙂 It's also frustrating not to be able to run a single test case when you want to.

Reply:
I remember it as a style of testing to RuboCop and others. Thank you for telling me the article of DAMP tests! I prefer this idea too 😃